src/flag_gems/ops/masked_fill.py

```diff
@@ -60,14 +41,7 @@ def masked_fill(inp, mask, value):
     inp = inp.contiguous()
     mask = mask.contiguous()
     expand_mask = mask.expand(inp.shape)
-    out = torch.empty_like(inp, dtype=inp.dtype, device=inp.device)
-
-    N = inp.numel()
-    if N == 0:
-        return out
-    grid = lambda meta: (triton.cdiv(N, meta["BLOCK_SIZE"]),)
-    masked_fill_kernel[grid](inp, expand_mask.to(torch.int), value, out, N)
-    return out
+    return masked_fill_kernel(inp, expand_mask.to(torch.int), value)

```

Comment:
PointwiseDynamicFunction can handle non-contiguous inputs, so there is not need to make inp, mask contiguous beforehand.
Also, there is no need to cast mask to integer type, boolean is also fine.